docs/source/datasets/index.rst

```diff
@@ -356,6 +363,7 @@ contractors or self-employed individuals, while a W-2 form is used for employees
 pseudopeople can generate a simulated version of the data collected by W-2 and 1099 forms.
 This is a yearly dataset, where the user-specified year is the **tax year** of the data.
 That is, the data for 2022 will be the result of tax forms filed in early 2023.
+Tax data can be generated for tax years 2019 through 2040 (inclusive).
```

Comment:
I'm definitely going to be confused about tax year vs filing year.  But I have confirmed that `psp.generate_taxes_w2_and_1099(year=2019)` returns some simulated data

Reply:
Yes, all the datasets (except SSA) start with 2019. But (unlike some other datasets) there is **no** tax data returned by `psp.generate_taxes_w2_and_1099(year=2041)` because that would have to be filed in 2042, which we didn't simulate.